tasks/main.yml

```diff
@@ -62,7 +67,6 @@
     owner: root
     group: root
     mode: 0644
-    backup: yes
```

Comment:
This would copy a the pervious version to with a new name in the same directory. Collectd will try lo load any file in the directory -- so any backups there can cause errors. Also, we don't need to back it up. So I removed it.